CMSIS/NN/Source/ConvolutionFunctions/arm_convolve_1_x_n_s8.c

```diff
@@ -41,23 +41,23 @@
  */
 
 /*
-   * 1xN s8 convolution function.
-   *
-   * Refer header file for details.
-   *
-   */
+ * 1xN s8 convolution function.
+ *
+ * Refer header file for details.
+ *
+ */
 
 arm_status arm_convolve_1_x_n_s8(const cmsis_nn_context* ctx,
                                  const cmsis_nn_conv_params* conv_params,
                                  const cmsis_nn_per_channel_quant_params* quant_params,
                                  const cmsis_nn_dims* input_dims,
-                                 const q7_t *input_data,
+                                 const q7_t* input_data,
```

Comment:
The pointer * alignment is off here..In some other files it looks ok. We would like to have it as const q7_t *input_data. 

could you try with PointerAlignment: Right option in the .clang-format file in the NN base folder?

Reply:
Done